lib/schema.js

```diff
@@ -337,7 +217,6 @@ class Schema {
   }
 }
 
-Schema.KIND = KIND;
 Schema.KIND_STORED = KIND_STORED;
```

Comment:
Minimum supported Node.js version by metaschema is 12.9 so we can just use `static` properties. https://node.green/#ES2022-features-static-class-fields-public-static-class-fields

Reply:
The reason for this is outdated `"eslint-config-metarhia"`. It depends on old eslint, so I got eslint error, which can not be disabled easily without update. I see no big issue with this approach. Syntax is OK. But we need to update our linter to support new ES features

Reply:
Can be done after eslint config update